src/games/strategy/engine/data/ChangeFactory.java

```diff
@@ -105,6 +105,7 @@ public static Change changeResourcesChange(final PlayerID player, final Resource
   }
 
   /** TODO: unused code, kept per @veqryn */
+  @SuppressWarnings("unused")
```

Comment:
I kept unused functions, because the involved logic was very complex and it would be a paint to reimplement all the stuff...